README.md

```diff
@@ -5,371 +5,88 @@ ADAM
 
 # Introduction
 
-ADAM is a genomics analysis platform with specialized file formats built using [Apache Avro][Avro], [Apache Spark][Spark] and [Apache Parquet][Parquet]. Apache 2 licensed. Some quick links: 
+ADAM is a library and command line tool that enables the use of [Apache
+Spark](https://spark.apache.org) to parallelize genomic data analysis across
+cluster/cloud computing environments. ADAM uses a set of schemas to describe
+genomic sequences, reads, variants/genotypes, and features, and can be used
+with data in legacy genomic file formats such as SAM/BAM/CRAM or VCF, as well
+as data stored in the columnar [Apache Parquet](https://parquet.apache.org)
+format. On a single node, ADAM provides competitive performance to optimized
+multi-threaded tools, while enabling scale out to clusters with more than a
+thousand cores. ADAM's APIs can be used from Scala, Java, Python, R, and SQL.
 
-* [Follow our Twitter account](https://twitter.com/bigdatagenomics/).
-* [Chat with ADAM developers in Gitter](https://gitter.im/bigdatagenomics/adam).
-* [Join our mailing list](http://bdgenomics.org/mail/).
-* [Checkout the current build status](https://amplab.cs.berkeley.edu/jenkins/view/Big%20Data%20Genomics/).
-* [Download official releases][releases].
-* [View our software artifacts on Maven Central](http://search.maven.org/#search%7Cga%7C1%7Corg.bdgenomics) ([…including snapshots](https://oss.sonatype.org/index.html#nexus-search;quick~bdgenomics)).
-* [Look at our CHANGES file](https://github.com/bigdatagenomics/adam/blob/master/CHANGES.md).
-
-## Why ADAM?
```

Comment:
I agree generally with DRY, but I think that this section of the README is necessary because it provides the "biologist relevant" info about why to use ADAM. Unlike a code sample or build instructions, it isn't going to go stale.